### ISSUE TITLE
the net is overfit in sst dataset

### ISSUE THREAD

**Issue by mathCrazyy** [mttk/rnn-classifier#1]
train loss is 0.1*, but the valid loss is 6*, if add the L2 and try to increase the dropout rate, but does'nt work, how should i do, exactly, the acc is 80%-82%, but i want to know is there some method to solve the overfit, and increase the epoch to get a new acc? thank you

**Comment by mathCrazyy** [comment]
sorry , valid loss is 0.6*.. just a writen wrong.

**Comment by mttk** [comment]
Hi, sorry for the slow reply:
In general for text classification you don't care as much about the loss as much as your evaluation metric of choice (accuracy / F1 / ...) so I feel this is not **very** worrying (you do expect the valid loss to be higher than train). The SST dataset is also very prone to overfitting (short inst. length; many high precision tokens indicating labels).

If you are still worried about the loss, I'm unsure what the best method to mitigate this would be for SST off the top of my head. The usual suspects are the # of model params (reducing hidden size, layer num, using smaller embeddings), learning rate (reducing until you see the desired effect), ...
